aws-bundle/NOTICE

```diff
@@ -4,3 +4,885 @@ Copyright 2017-2023 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.15
+
+Apache Commons Codec
+Copyright 2002-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
+contains test data from http://aspell.net/test/orig/batch0.tab.
+Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
+
+===============================================================================
+
+The content of package org.apache.commons.codec.language.bm has been translated
+from the original php source code available at http://stevemorse.org/phoneticinfo.htm
+with permission from the original authors.
+Original source copyright:
+Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: commons-logging  Name: commons-logging  Version: 1.2
+
+Apache Commons Logging
+Copyright 2003-2014 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
I hate to make this more annoying, but we _don't_ want to include a NOTICE if it contains only ASF boilerplate like this. We can omit it and should be cause we need to produce minimal NOTICE content.

Reply:
OK, this is done.